mmv1/third_party/terraform/services/gemini/iam_gemini_repository_group_test.go.tmpl

```diff
@@ -0,0 +1,226 @@
+package gemini_test
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccGeminiRepositoryGroupIamBinding(t *testing.T) {
+	location := "us-central1"
+	codeRepositoryIndexId := acctest.BootstrapSharedCodeRepositoryIndex(t, "basic", location, "", map[string]string{"ccfe_debug_note": "gcloud_e2e_do_not_delete"})
```

Comment:
`gcloud_e2e_do_not_delete` – this should be `terraform_e2e_do_not_delete` if this is planned to be static

Reply:
After set of test cases this should be removed too, so replaced with `terraform_e2e_should_be_deleted`. This should only not removed when tests are running